galaxy/api/internal/serializers/collections.py

```diff
@@ -62,10 +62,10 @@ class Meta:
 
 
 class VersionSummarySerializer(serializers.ModelSerializer):
-    '''
+    """
     Returns summary information for a collection version. Returning all of a
     collection's contents in a list is too much data to surface.
-    '''
+    """
```

Comment:
Let's give this the PEP standard format of a single line, a blank line, a paragraph. First line with the `"""`